proto/sf/substreams/rpc/v2/service.proto

```diff
@@ -130,6 +130,37 @@ message OutputDebugInfo {
 
 message ModulesProgress {
   repeated ModuleProgress modules = 1;
+  repeated Job running_jobs = 2;
+  repeated ModuleStats modules_stats = 3;
+}
+
+message Job {
```

Comment:
We've phased out `Job` everywhere, used `WorkUnit` instead.. perhaps this should look similar. Is it refering to the same work.Unit ?

Reply:
Our documentation mentions "tier2 jobs" or "running jobs on tier2", and we have flags like `X-Sf-Substreams-Parallel-Jobs`

I think it is easy enough to grasp the meaning of a 'job' in that context. The back-end implementation uses WorkUnit to determine a **segment** and a **stage**, which is the unit that can be made into a job. It is not exactly the same meaning, and "work unit" everywhere would make documentation cumbersome.